src/lib/djerba/helpers/pwgs_cardea_helper/helper.py

```diff
@@ -44,14 +44,14 @@ def get_cardea(self, requisition_id, cardea_url):
             raise MissingCardeaError(msg)
         else:
             requisition_json = json.loads(r.text)
-            assay_name = requisition_json['assayName'].split("-")[0].strip().upper()
-            cases = requisition_json['cases']
-            if len(cases) > 1 or len(cases) < 1: # only one case expected for clinical 
-                msg = "{0} case(s) were found. Only 1 case is expected".format(len(cases))
+            
+            if len(requisition_json) > 1 or len(requisition_json) < 1: # only one case expected per requisition for clinical 
+                msg = "{0} case(s) were found. Only 1 case is expected".format(len(requisition_json))
                 self.logger.error(msg)
                 raise ValueError(msg)
             else:
-                case = requisition_json['cases'][0]
+                case = requisition_json[0]
+                assay_name = case['assayName'].split("-")[0].strip().upper() 
```

Comment:
Could wrap this line in a try/except in case we ever meet Cardea output which isn't in the expected format:

```
try:
    assay_name = case['assayName'].split("-")[0].strip().upper() 
except KeyError, IndexError as err:
    msg = "Unexpected format for Cardea results: {0}".format(err)
    self.logger.error(msg)
    raise ValueError(msg) from err
```